sdk/cognitiveservices/cognitiveservices-luis-authoring/src/lUISAuthoringClient.ts

```diff
@@ -34,8 +30,8 @@ class LUISAuthoringClient extends LUISAuthoringClientContext {
    * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param [options] The parameter options
    */
-  constructor(endpoint: string, credentials: msRest.ServiceClientCredentials, options?: msRest.ServiceClientOptions) {
-    super(endpoint, credentials, options);
+  constructor(credentials: msRest.ServiceClientCredentials, endpoint: string, options?: msRest.ServiceClientOptions) {
+    super(credentials, endpoint, options);
```

Comment:
Was this change made manually or did the code generator do this?

Reply:
It's the code generator.

Reply:
Oh perfect, then one of the issues I had with luis-authoring regarding the order of the arguments being wrong is fixed by the code generator!